lint/base_linter/python_linter.py

```diff
@@ -39,23 +39,9 @@ class PythonLinter(linter.Linter):
     """
 
     @classmethod
-    @lru_cache(maxsize=None)
-    def can_lint(cls, syntax):
-        """Determine optimistically if the linter can handle the provided syntax."""
-        can = False
-        syntax = syntax.lower()
-
-        if cls.syntax:
-            if isinstance(cls.syntax, (tuple, list)):
-                can = syntax in cls.syntax
-            elif cls.syntax == '*':
-                can = True
-            elif isinstance(cls.syntax, str):
-                can = syntax == cls.syntax
-            else:
-                can = cls.syntax.match(syntax) is not None
-
-        return can
+    def can_lint(cls):
+        """Assume the linter can lint."""
+        return True
```

Comment:
I know this is technically not a part of this PR, but why override `can_lint` at all?

Reply:
I don't want the base classes can_lint. 🤔

Reply:
Ah right, the specialized linters don't rely on the global exeutable and instead have this context sensitive executable path thing.